bootstrap.sh

```diff
@@ -252,7 +255,7 @@ if [ -d /etc/appscale/certs ]; then
 
 
         # Let's upgrade the repository: if GIT_TAG is empty we are on HEAD.
-        if [ -n "${GIT_TAG}" ]; then
+        if [ -n "${GIT_TAG}" ] && [ "${UPDATE_REPO}" = "Y" ]; then
```

Comment:
Some style guides prefer avoiding `-a` for clarity: https://github.com/koalaman/shellcheck/wiki/SC2166